parking/backend/sensor_server/rest_server.py

```diff
@@ -0,0 +1,81 @@
+import json
+from tornado import web
+from parking.shared.rest_models import (ParkingLot, ParkingLotCreationResponse, ParkingLotAvailableMessage,
+                                        ParkingLotPriceMessage, InvalidLotFieldsError, InvalidLotIDError,
+                                        InvalidLotAvailableError, RESTError, InvalidLotPriceError)
+from parking.shared.util import serialize_model
+from parking.backend.db.dbaccess import DbAccess
+
+
+class ParkingLotsHandler(web.RequestHandler):
+    def initialize(self, dba: DbAccess) -> None:
+        self.dba = dba
+
+    async def post(self):
+        try:
+            lot = ParkingLot(**json.loads(self.request.body))
+        except (TypeError, ValueError) as err:
+            self.write(serialize_model(self._create_error_response(err)))
+        else:
+            pid = await self.dba.insert_parking_lot(lot)
+            self.write(serialize_model(ParkingLotCreationResponse(id=pid)))
+
+    def _create_error_response(self, err: Exception) -> ParkingLotCreationResponse:
+        if isinstance(err, ValueError):
+            return ParkingLotCreationResponse(error=InvalidLotFieldsError(str(err)))
+        else:
+            return ParkingLotCreationResponse(error=InvalidLotFieldsError('Invalid or missing fields'))
+
+
+class IndividualLotHandler(web.RequestHandler):
+    def initialize(self, dba: DbAccess) -> None:
+        self.dba = dba
+
+    def prepare(self):
+        if self.request.headers.get("Content-Type", "").startswith("application/json"):
+            self.json_args = json.loads(self.request.body)
+        else:
+            self.json_args = None
+
+    def _create_error_response(self, err: Exception, parse_err_cls: RESTError) -> RESTError:
+        if isinstance(err, ValueError):
+            return parse_err_cls(str(err))
+        else:
+            return parse_err_cls('Invalid or missing fields')
+
```

Comment:
Doesn't need to be added now but it would be nice to be able to query this stuff over the REST api with `get` as well (which might also make integration testing a bit easier).

Reply:
Yeah, that'd definitely be helpful for testing - I can open an issue for it and we can discuss the kind of data that would be useful to pull out from the API using GET.